src/Message/RefundRequest.php

```diff
@@ -50,6 +50,16 @@
  */
 class RefundRequest extends AbstractRequest
 {
+    public function getRefundApplicationFee()
+    {
+        return $this->getParameter('refundApplicationFee');
+    }
+
+    public function setRefundApplicationFee($value)
+    {
+        return $this->setParameter('refundApplicationFee', $value);
+    }
+
```

Comment:
Can we get some doc blocks on these functions? It wasn't clear to me at first that you're passing in a boolean and I worry that other people may have such a problem?